rpcserver.go

```diff
@@ -1068,7 +1069,8 @@ func allowCORS(handler http.Handler, origins []string) http.Handler {
 // address to a specified output value to be sent to that address.
 func (r *rpcServer) sendCoinsOnChain(paymentMap map[string]int64,
 	feeRate chainfee.SatPerKWeight, minConfs int32,
-	label string) (*chainhash.Hash, error) {
+	label string,
+	strategy wallet.CoinSelectionStrategy) (*chainhash.Hash, error) {
 
 	outputs, err := addrPairsToOutputs(paymentMap, r.cfg.ActiveNetParams.Params)
 	if err != nil {
```

Comment:
I think you would want to add the strategy to the dry run here as well, https://github.com/lightningnetwork/lnd/blob/ea2d5373fa4a0d76c6990302ca5d5f864d858c88/rpcserver.go#L1082